### PR TITLE
#34640: CLN: remove 'private_key' and 'verbose' from gbq

### LINKED ISSUE #34640: CLN: remove `private_key` and `verbose` from gbq
Since `verbose` and `private_key` have been deprecated since https://github.com/pandas-dev/pandas/pull/30200, we should also remove both from function since they are not used at all? see them in `read_gbq` and `to_gbq`, e.g.

https://github.com/pandas-dev/pandas/blob/24857a2844be9ee6f604d93ea18301f587751f0a/pandas/io/gbq.py#L33

https://github.com/pandas-dev/pandas/blob/24857a2844be9ee6f604d93ea18301f587751f0a/pandas/io/gbq.py#L34

### PR DESCRIPTION
**Pull request by parkdj1**
- [x ] closes #34640 
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

Hello, I am new to contributing to open source, so thank you in advance for any corrections and suggestions. Please let me know if I need to do anything else for this ticket!

I removed the code containing both 'private_key' and 'verbose' from the gbq.py file.

### CODE REVIEW

**Comment by WillAyd** [comment]
@tswast

**Comment by charlesdong1991** [comment]
since those two kwargs weren't removed along with #30200 , also cc @jbrockmendel , just in case this was made on purpose by him.

**Review by tswast — APPROVED** [review]

**Review by jreback — CHANGES_REQUESTED** [review]
can you add a note in other api changes section for 1.1 that these keywords were removed, also please merge master and ping on green.

File: doc/source/whatsnew/v1.1.0.rst [hunk 1]
```diff
@@ -379,7 +379,7 @@ Other API changes
 - ``loc`` lookups with an object-dtype :class:`Index` and an integer key will now raise ``KeyError`` instead of ``TypeError`` when key is missing (:issue:`31905`)
 - Using a :func:`pandas.api.indexers.BaseIndexer` with ``count``, ``min``, ``max``, ``median``, ``skew``,  ``cov``, ``corr`` will now return correct results for any monotonic :func:`pandas.api.indexers.BaseIndexer` descendant (:issue:`32865`)
 - Added a :func:`pandas.api.indexers.FixedForwardWindowIndexer` class to support forward-looking windows during ``rolling`` operations.
--
+- Removed ``private_key`` and ``verbose`` from gbq (:issue:`34654`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.1.0.rst:382]: can you co-locate this with the other issue where we removed a keyword from gbq; also pls use the references to read_gbq (see the other issue)

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by charlesdong1991** [comment]
@parkdj1 

hi, you need to rebase and resolve the conflicts in the two files listed above, otherwise, your commit won't get through and make CI run.

**Comment by alimcmaster1** [comment]
@parkdj1 - Thanks for the PR, can you address the test failures and fix up the commits seems to be a few unrelated changes.

**Comment by parkdj1** [comment]
> Hello @parkdj1! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:
> 
> There are currently no PEP 8 issues detected in this Pull Request. Cheers! 🍻
> 
> ##### Comment last updated at 2020-06-29 00:35:21 UTC

Awesome, thank you!

**Comment by alimcmaster1** [comment]
@parkdj1 is this still active. Ref my comments above:  https://github.com/pandas-dev/pandas/pull/34654#issuecomment-647164000

Also whatsnew will be 1.2 now.

Thanks

File: doc/source/whatsnew/v1.2.0.rst [hunk 2]
```diff
@@ -281,7 +281,8 @@ ExtensionArray
 
 Other
 ^^^^^
--
+
+- Removed ``private_key`` and ``verbose`` from :func:`pandas.read_gbq` (:issue:`34654` :issue: `30200`)
```
Comment (charlesdong1991) [inline doc/source/whatsnew/v1.2.0.rst:285]:
```suggestion
- Removed ``private_key`` and ``verbose`` from :func:`read_gbq` (:issue:`34654` :issue:`30200`)
```

**Review by charlesdong1991 — COMMENTED** [review]
since needs to resolve conflict, a minor comment ^^

File: doc/source/whatsnew/v1.2.0.rst (hunk 2 above)
Comment (WillAyd) [reply doc/source/whatsnew/v1.2.0.rst:285]: Great - thanks for the callout

Comment (charlesdong1991) [reply doc/source/whatsnew/v1.2.0.rst]: sorry, seems there is still a space between `:issue:` and `30200` now? 🤔

File: doc/source/whatsnew/v1.2.0.rst [hunk 3]
```diff
@@ -340,6 +340,8 @@ ExtensionArray
 
 Other
 ^^^^^
+
+- Removed ``private_key`` and ``verbose`` from :func:`read_gbq` (:issue:`34654` :issue:`30200`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.2.0.rst:344]:
```suggestion
- Removed ``private_key`` and ``verbose`` from :func:`read_gbq` as not longer supported in `pandas-gbq` (:issue:`34654` :issue:`30200`)
```

File: doc/source/whatsnew/v1.2.0.rst [hunk 4]
```diff
@@ -340,6 +340,8 @@ ExtensionArray
 
 Other
 ^^^^^
+
```
Comment (jreback) [inline doc/source/whatsnew/v1.2.0.rst:343]: can you move this to the I/O section

**Review by jreback — CHANGES_REQUESTED** [review]
minor comments, pls merge master and ping on green. I don't think we have any remaining tests for these right?

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @parkdj1

**Pull request merged**